sprint2-html/Index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CV Digital</title>
+    <style>
+      a {
+        text-decoration: none;
+      }
+      h2 {
+        font-size: 25px;
+        color: blue;
+        margin: 25px 0 0;
+      }
+    </style>
+  </head>
+  <body>
+    <div>
+      <h1>Alexandra <span style="color: blue">Giurgiu</span></h1>
+      <p>
+        Rastatt, 76437 &#9679;
+        <a href="tel:+4915145570968"
+          ><span style="color: black">+4915145570968</span></a
+        >
+      </p>
+      <p>
+        <b
+          >EmailID &#9679;
+          <a href="mailto:topirceanalexandra@yahoo.com"
+            ><span style="color: blue">topirceanalexandra@yahoo.com</span></a
+          ></b
+        >
+      </p>
+    </div>
```

Comment:
```suggestion
    </div>
    
```

(opt) - putem adauga un rand nou intre blocuri ca sa se citeasca codul mai usor